doc/source/workflow/overview.md

```diff
@@ -0,0 +1,208 @@
+- **Non-Reusable Model Servers**: Specialised server meant to serve a single model.
+  Does not require the central repository but requires a build of a new image for every model.
+
+
+![](../images/model-servers.svg)
+
+Read more about our pre-packaged `Model Servers` on their dedicated documentation pages:
+- [MLflow Server](../servers/mlflow.html)
+- [SKLearn Server](../servers/sklearn.html)
+- [Tensorflow Server](../servers/tensorflow.html)
+- [XGBoost Server](../servers/xgboost.html)
+
+Read how to build your own pre-packaged model server [here](../servers/custom.html).
+
+## Language Wrappers
+
+Language wrappers allows Seldon Core users to build reusable and non-reusable model servers.
+As you will see, the whole process is very simple and requires user to only define logic that
+loads models and perform inference prediction as well as required runtime dependencies.
+
+![](../images/language-wrappers-1.svg)
+
+
+Model loading and inference logic is defined in `Model.py` file:
+```python
+class Model:
+  def __init__(self):
+    # custom logic that prepare model
+    self._model = ...
+
+  def predict(self, features, names=[], meta=[]):
+    # custom inference logic
+    return self._model.predict(...)
+```
+
+If `Model`'s `__init__` method does not contain logic to load models this would be the non-reusable model server.
+
+The `seldon-core-microservice` Python wrapper can be used to turn `Model.py` into a fully operational microservice.
+
+![](../images/language-wrappers-2.svg)
+
+To complete containerisation process you need two more components:
+- `requirements.txt` file that describes your runtime dependencies
+- `.s2/environment` file that describes your microservice (api and model type)
+
+Once these are in place you can use a simple s2i command
+```bash
+s2i build . seldonio/seldon-core-s2i-python3:1.1.0 model:0.1
+```
+to create ready to use Docker image.
+
+Read more about Python `Language Wrapper` on its dedicated documentation [page](../python/index.html).
+
+
+## Seldon Deployment CRD
+
+Seldon Deployment CRD is the real strength of Seldon Core. It allows you to easily deploy your inference model to the Kubernetes cluster and handle some real production traffic!
+
+The manifest file that you write is very powerful yet simple.
+You can easily define what models do you want in your deployment and how they are connected in the inference graph.
+
+
+![](../images/seldondeployment-crd.svg)
+
+Read more about Seldon Deployment CRD on its dedicated documentation [page](../reference/seldon-deployment.html).
+
+
+## Seldon Core Operator
+
+Seldon Core `Operator` is what controls your `Seldon Deployments` in the `Kubernetes` cluster.
+It reads the CRD definition of `Seldon Deployment` resources applied to the cluster and takes
+care that all required components like `Pods` and `Services` are created.
+
+It works according to the common Kubernetes operator pattern - in a continues loop it:
```

Comment:
NIce - may be worth outlining some of the features it adds on top (there was a list you created which you added in the slide)

Reply:
Hmm... I thought that if it is on the slide it is enough? 
![image](https://user-images.githubusercontent.com/10928117/86608795-e83adb80-bfa2-11ea-9edb-3343b59ab58a.png)